src/lib/party.ts

```diff
@@ -836,6 +860,57 @@ function addMissingSignatures(
   };
 }
 
+type PartiesProved = {
+  feePayer: FeePayer;
+  otherParties: (Party & { authorization: Control | LazySignature })[];
+};
+
+async function addMissingProofs(parties: Parties): Promise<PartiesProved> {
```

Comment:
When does this get called or is intended to be called?

Reply:
This is called under the hood by `transaction.prove()`